culture/review_culture.md

```diff
@@ -147,3 +172,26 @@ doing this, you should ask: in what ways is doing it my way materially better?
 What benefits come from the changed approach? If you don't have clear answers
 to these questions, don't ask for the change.
 
+### Take every measure to include contributors
+
+It could be the case that your thinking has changed since a review has started,
+and the change is no longer logical or necessary for the codebase. It might
+also be the case that an individual is new to contributing on a platform,
+and isn't quite sure how to best help. In both cases, as a maintainer you should
+take every measure to engage the contributors. For example, although this would
+be a respectful response:
+
+> Thank you for your contribution! We've decided to go in another direction, so unfortunately we will not be integrating the change.
+
+It ends the participation right away. Instead, you might consider
+a statement that delivers the same message, but further engages the contributor:
+
+> Thank you for your contribution! We've decided to go in another direction, and although we cannot use the changes here, we are interested in your thoughts on the idea here. Would you like to help?
```

Comment:
I have a slight preference for the example you gave in your original comment, "Would you be interested in working on [well-scoped task]?", as a better template compared to "we are interested in your thoughts on the idea here". Not requesting a change, just a thought.

Reply:
Fixed!